src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -372,6 +380,11 @@ export class StudyViewPageStore
     @observable numberOfVisibleColorChooserModals = 0;
     @observable userGroupColors: { [groupId: string]: string } = {};
 
+    chartsBinMethod: { [chartKey: string]: BinMethodOption } = {};
```

Comment:
this shouldn't be observable?

Reply:
@adamabeshouse  I have refactored the use of the store for the CustomBinsModal. Now the fields are observable. You can find the changes in the last commit.